setup.py

```diff
@@ -454,10 +454,6 @@ def _quote_build_define(argument):
         ("GRPC_ENABLE_FORK_SUPPORT", 1),
     )
 
-# Fix for multiprocessing support on Apple devices.
```

Comment:
My mistake, @yijiem. This will bring back some Python multiprocessing issues that our own tests are not yet sensitive to. Please leave the poller disabled.

Reply:
Done.